app/helpers/katello/concerns/smart_proxy_helper_extensions.rb

```diff
@@ -0,0 +1,51 @@
+module Katello
+  module Concerns
+    module SmartProxyHelperExtensions
+      def disks(storage)
+        mount_points = {}
+        disks = []
+        storage.each do |_name, values|
+          mount = values['mounted']
+          mount_points[mount].nil? ? mount_points[mount] = [values['path']] : mount_points[mount] << values['path']
+          values['header'] = "#{mount_points[mount].to_sentence} (on #{values['filesystem']})"
+          values['available_percent'] = available_percent(values['percent'])
+          values['size_status'] = storage_warning(values['available'])
+          values['total'] = kb_to_actual(values.delete('1k-blocks'))
+          values['used'] = kb_to_actual(values['used'])
+          values['available'] = kb_to_actual(values['available'])
+          disks << values
+        end
+        disks.group_by { |h| h['mounted'] }.map { |_, hs| hs.reduce(:merge) }
+      end
+
+      def boolean_to_icon(boolean)
+        boolean = Foreman::Cast.to_bool(boolean)
```

Comment:
getting an error, i think you need to prepend '::'  onto Foreman::Cast

we might should try to remove the Katello::Foreman module at some point :)